CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,121 @@
+## TL;DR
+
+Be nice. Provide and accept constructive feedback. Avoid spamming, abusive, trolling, and otherwise unacceptable behaviour. Repeat violations may result in a permanent ban.
```

Comment:
```suggestion
Be nice. Provide and accept constructive feedback. Avoid spamming, abusive, trolling, and otherwise unacceptable behavior. Repeat violations may result in a permanent ban.
```